arangodb-net-standard.Test/Serialization/JsonNetApiClientSerializationTest.cs

```diff
@@ -192,6 +223,33 @@ public void Serialize_ShouldNotCamelCaseParams_WhenSerializingPostTransactionBod
             Assert.DoesNotContain("dontCamelCaseKey", jsonString);
         }
 
+
+        [Fact]
+        public void Serialize_ShouldCamelCaseParams_WhenSerializingPostTransactionBody()
+        {
+            var body = new PostTransactionBody
+            {
+                Params = new Dictionary<string, object>
+                {
+                    ["DontCamelCaseKey"] = new { DontCamelCaseMe = true }
```

Comment:
Nitpick: the name should be `CamelCaseMe` if this must be camel cased by the test, otherwise it's a bit confusing 🤔

Reply:
Fixed the name.